README.md

```
@@ -19,158 +17,81 @@ BDD, API testing and CI/CD integration.
-- [ ] Retry failed tests
-- [ ] Slack notifications
 - [x] AI code reviews on pull requests
+- [ ] Allure test reporting
+- [ ] REST Assured + WireMock API testing
+- [ ] CartPage and cart tests
+- [ ] CheckoutPage and end-to-end checkout tests
+- [ ] Test grouping for smoke and regression suites
+- [ ] Environment switching
+- [ ] Cross-browser test matrix
+- [ ] Docker + Selenium Grid
+- [ ] Retry failed tests
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)